go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/cert-manager/csi-driver
 
 go 1.22.0
+toolchain go1.23.2
```

Comment:
Not sure why dependabot added this. Let's see if the tests pass.

Reply:
The tests did not pass:

> go: updates to go.mod needed; to update it:
> 	go mod tidy
> make: *** [make/_shared/oci-build/01_mod.mk:52: oci-build-manager] Error 1
-- https://prow.infra.cert-manager.io/view/gs/cert-manager-prow-artifacts/pr-logs/pull/cert-manager_csi-driver/307/pull-cert-manager-csi-driver-e2e/1842157677040373760

Dependabot added the `toolchain` directive.
I had to run `make go-tidy` which bumped the go directive.
Probably caused by: https://github.com/cert-manager/csi-lib/pull/67
 * https://github.com/cert-manager/csi-lib/pull/67